hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceManager.java

```diff
@@ -1070,4 +1070,21 @@ MetricsReplicationGlobalSourceSource getGlobalMetrics() {
   ReplicationQueueStorage getQueueStorage() {
     return queueStorage;
   }
+
+  boolean addTotalBufferUsed(long size) {
```

Comment:
Please add comments to mention the meaning of the return value.

Reply:
After checking the later code, I think here we'd better introduce two methods, one is acquire, which returns a boolean, the other is release, which returns nothing. And they could both call a private method, where we change the totalBufferUsed, and record the new usage in metrics.
This will be much clearer.

Reply:
@Apache9 ，thanks for  suggestion, I would make it more clearer.